InvenTree/report/api.py

```diff
@@ -295,13 +296,13 @@ def filter_queryset(self, queryset):
 
             valid_report_ids = set()
 
-            for report in queryset.all():
+            for qreport in queryset.all():
```

Comment:
Comment: Without changing the imports this variable rename would be unnecesarry

Reply:
Correct. But, see above.